ui/raidboss/data/06-ew/raid/p11n.ts

```diff
@@ -1,14 +1,195 @@
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Line Cleave -> In',
+        },
+      },
+    },
+    {
+      id: 'P11N Divisive Ruling Out',
+      type: 'StartsUsing',
+      netRegex: { id: '81B3', source: 'Themis', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Line Cleave -> Out',
+          de: 'Linien AoE -> Raus',
+          fr: 'AoE en ligne -> Extérieur',
+          ja: '直線AoE -> 離れる',
+          cn: '直线AoE -> 远离',
+          ko: '직선 장판 -> 바깥으로',
+        },
+      },
+    },
+    {
+      id: 'P11N Dark Discord',
+      type: 'GainsEffect',
+      netRegex: { effectId: 'DE4' },
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Get in light puddle',
+        },
+      },
+    },
+    {
+      id: 'P11N Light Discord',
+      type: 'GainsEffect',
+      netRegex: { effectId: 'DE3' },
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Get in dark puddle',
+        },
+      },
+    },
+    {
+      id: 'P11N Dineis',
+      type: 'StartsUsing',
+      netRegex: { id: '8725', source: 'Themis', capture: false },
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Bait puddles x2',
+        },
+      },
+    },
+    {
+      id: 'P11N Dismissal Ruling Dynamo',
+      type: 'StartsUsing',
+      netRegex: { id: '81FB', source: 'Themis', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Knockback -> get in',
+        },
+      },
+    },
+    {
+      id: 'P11N Dismissal Ruling Chariot',
+      type: 'StartsUsing',
+      netRegex: { id: '81FA', source: 'Themis', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Knockback -> stay out',
+        },
+      },
+    },
+    {
+      id: 'P11N Dike Collect',
+      type: 'HeadMarker',
+      netRegex: { id: '00DA' },
+      run: (data, matches) => {
+        data.busterTargets ??= [];
+        data.busterTargets.push(matches.target);
+      },
+    },
+    {
+      id: 'P11N Dike Call',
+      type: 'HeadMarker',
+      netRegex: { id: '00DA', capture: false },
+      delaySeconds: 0.5,
+      suppressSeconds: 10,
+      infoText: (data, _matches, output) => {
+        const target1 = data.busterTargets?.[0] || '???';
+        const target2 = data.busterTargets?.[1] || '???';
+        return output.text!({ target1: target1, target2: target2 });
+      },
+      outputStrings: {
+        text: {
+          en: 'Tank buster: ${target1}, ${target2}',
```

Comment:
Hmm.  What about just "tank buster on you" (alert) vs "tank busters" (info, for the people it's not on)? I think that's what we've done elsewhere.